tests/cypress/tests/standalone/readonly_navigation_test.ts

```diff
@@ -0,0 +1,38 @@
+import DashTable from 'cypress/DashTable';
+import DOM from 'cypress/DOM';
+import Key from 'cypress/Key';
+
+import { AppMode } from 'demo/AppMode';
+
+Object.values([AppMode.ReadOnly]).forEach(mode => {
+    describe(`navigate (readonly), mode=${mode}`, () => {
+        beforeEach(() => {
+            cy.visit(`http://localhost:8080?mode=${mode}`);
+            DashTable.toggleScroll(false);
+        });
+
+        describe('with keyboard', () => {
+            beforeEach(() => {
+                DashTable.getCell(3, 1).click();
+            });
+
+            // Note: Dropdown cell is a label when readonly. Making a special
+            // version of the test for the read only table.
+            describe('into a dropdown cell', () => {
+                beforeEach(() => {
+                    DashTable.getCell(3, 5).click();
+                });
+
+                it('can move', () => {
+                    DOM.focused.type(Key.ArrowRight);
+
+                    DashTable.getCell(3, 6).should('have.class', 'focused');
+                    DOM.focused.type(Key.ArrowLeft, { force: true });
```

Comment:
Specifically this line https://github.com/plotly/dash-table/pull/293/files#diff-63e03da9cceb32b9aaddc0be5ea11e01R98 is gone